extra/twig-extra-bundle/Tests/Fixture/Kernel.php

```diff
@@ -24,11 +24,17 @@ public function registerBundles(): iterable
 
     protected function configureContainer(ContainerBuilder $c, LoaderInterface $loader): void
     {
-        $c->loadFromExtension('framework', [
+        $config = [
             'secret' => 'S3CRET',
             'test' => true,
-        ]);
-
+            'router' => ['utf8' => true],
+            'http_method_override' => false,
+        ];
+        if (Kernel::MAJOR_VERSION >= 6 &&  Kernel::MINOR_VERSION >= 2) {
```

Comment:
* We don't need this setting in Symfony 7, but this condition would be evaluate to `true` again once Symfony 7.2 is available.
* Since you've bumped to `^6.4`, we don't need to check the minor version anymore.

```suggestion
        if (Kernel::MAJOR_VERSION === 6) {
```

Reply:
Fixed